creme/tree/_split_criterion/variance_reduction_split_criterion.py

```diff
@@ -0,0 +1,50 @@
+from .base_split_criterion import SplitCriterion
+
+
+class VarianceReductionSplitCriterion(SplitCriterion):
+    """Variance Reduction split criterion.
+
+    Often employed in cases where the target variable is continuous (regression tree),
+    meaning that use of many other metrics would first require discretization before being applied.
+
+    `Wikipedia entry <https://en.wikipedia.org/wiki/Decision_tree_learning#Variance_reduction>`_
```

Comment:
Change link format to markdown

Reply:
Changed in commit [2a55a14](https://github.com/creme-ml/creme/pull/362/commits/2a55a14fa6f2b98d3160bd46f55f85d9fb6f272c).